pkg/protocol/xprotocol/dubbo/decoder.go

```diff
@@ -194,6 +195,11 @@ func getServiceAwareMeta(ctx context.Context, frame *Frame) (map[string]string,
 
 		// decode the attachment to get the real service and group parameters
 		if !matched && (listener == EgressDubbo || listener == IngressDubbo) {
+			field, err = decoder.Decode()
+			if err != nil {
+				return nil, fmt.Errorf("[xprotocol][dubbo] decode dubbo argsTypes error, %v", err)
```

Comment:
please change to `[xprotocol][dubbo] decode dubbo argument types error, %v`

Reply:
already change it.